lib/dry/component/container.rb

```diff
@@ -176,9 +177,23 @@ def self.imports
         @imports ||= {}
       end
 
+      def self.imported
+        @imported ||= {}
+      end
+
       private
 
+      def self.imported?(name)
+        imported.key?(name)
+      end
+
+      def self.import_containers
+        imports.each { |ns, container| import_container(ns, container) }
+      end
```

Comment:
Oh wait. Nevermind. The name of the meth confused me. Maybe should be sth like `load_import_containers` or something